tests/modules/nf-scil/betcrop/fslbetcrop/nextflow.config

```diff
@@ -3,7 +3,13 @@ process {
     publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
```

Comment:
Move this inside the `"BETCROP_FSLBETCROP"` selector, so the `LOAD_TEST_DATA` workflow outputs don't get included in the `test.yml`